README.md

```diff
@@ -35,12 +35,38 @@ Email: info@iliosproject.org
 
 # Development of Ilios
 
-You can build a run a local version of Ilios easily using [docker](https://www.docker.com/).
-There are convenient clients for Mac and Windows and instructions Linux.
+The Ilios API is a PHP application which required a MySql server.  The easiest way to get a working
+MySql database running is by using the ilios/mysql-demo [docker](https://www.docker.com/) image.
+After installing docker you can start a demo database by running:
 
-After you have installed docker on your machine you can startup a local version by changing into the Ilios directory and running:
 ```bash
-docker-compose up
+docker run -d --name db -p 3306:3306 ilios/mysql-demo
 ```
 
-You will now be able to access ilios at http://localhost:8000
+Now modify your `/app/config/parameters.yml` to connect to this database:
```

Comment:
Then, modify your `/app/config/parameters.yml` configuration file to connect to this database: